version.py

```diff
@@ -1 +1 @@
-__version__ = "0.10.0"
+__version__ = "0.11.0"
```

Comment:
"1.0.0" 🚀

Reply:
hehe Im willing to say pre-release, and then try it out, and then bumping if we think its 👍